funnel/forms/space.py

```diff
@@ -77,6 +69,21 @@ def validate_bg_color(self, field):
             raise forms.ValidationError("Please enter a valid color code")
 
 
+class ProposalSpaceTransitionForm(forms.Form):
+    transition = forms.SelectField(__("Status"), validators=[
+                                   forms.validators.DataRequired()])
+
+    def set_queries(self):
+        """
+        value: transition method name
+        label: transition object itself
+        We need the whole object to get the additional metadata in templates
+        """
+        self.transition.choices = sorted([
+            (name, transition) for name, transition in self.edit_obj.state.transitions().items()
```

Comment:
Why can't this be just `self.transition.choices = sorted(self.edit_obj.state.transitions().items())`